esp/bindings/http/platform/httpbinding.cpp

```diff
@@ -2027,6 +2027,10 @@ int EspHttpBinding::onStartUpload(IEspContext &ctx, CHttpRequest* request, CHttp
     {
         me->append(*MakeStringExceptionDirect(-1, "Unknown Exception"));
     }
+
+    //There is an exception. So close socket connection right away to avoid more file upload.
```

Comment:
@wangkx the code is good, the comment better be 

//There is an exception. So close socket connection right after sending the response to avoid more file upload.

The reason is that the response (that contains the exception) will be sent out, even though the client may not read it as it is still busy sending the request...

Reply:
@mayx good comment. I just made the change.